File: src/test/java/test/SearchDepartmentMembersTest.java
package test;
import cn.authing.sdk.java.client.ManagementClient;
import cn.authing.sdk.java.dto.*;
import cn.authing.sdk.java.model.ManagementClientOptions;
import cn.authing.sdk.java.util.JsonUtils;

import java.util.Collections;



public class SearchDepartmentMembersTest {

    private static final String ACCESS_KEY_ID = "AUTHING_ACCESS_KEY_ID";
    private static final String ACCESS_KEY_SECRET = "AUTHING_ACCESS_KEY_SECRET";

    public static void main(String[] args) throws Throwable {
        ManagementClientOptions clientOptions = new ManagementClientOptions();
        clientOptions.setAccessKeyId(ACCESS_KEY_ID);
        clientOptions.setAccessKeySecret(ACCESS_KEY_SECRET);
        ManagementClient client = new ManagementClient(clientOptions);

        SearchDepartmentMembersDto request = new SearchDepartmentMembersDto();
        request.setOrganizationCode("organizationCode_2608");
        request.setDepartmentId("departmentId_6637");
        request.setKeywords("keywords_6673");
        request.setPage(0);
        request.setLimit(0);
        request.setDepartmentIdType("departmentIdType_4382");
        request.setIncludeChildrenDepartments(Boolean.TRUE);
        request.setWithCustomData(Boolean.TRUE);
        request.setWithIdentities(Boolean.TRUE);
        request.setWithDepartmentIds(Boolean.TRUE);

        UserPaginatedRespDto response = client.searchDepartmentMembers(request);
        System.out.println(JsonUtils.serialize(response));
    }

}

File: src/test/go/test/SearchDepartmentMembersTest.go
package main

import (
"authing-go-sdk/client"
"authing-go-sdk/dto"
"fmt"
)

func main() {
    options := client.ManagementClientOptions{
        AccessKeyId:     "AUTHING_USERPOOL_ID",
        AccessKeySecret: "AUTHING_USERPOOL_SECRET",
    }
    var err error
    client, err := client.NewClient(&options)
    if err != nil {
        panic(err)
    }
    request := dto.SearchDepartmentMembersDto{
        OrganizationCode: "organizationCode_2963",
        DepartmentId: "departmentId_3706",
        Keywords: "keywords_8339",
        Page: 0,
        Limit: 0,
        DepartmentIdType: "departmentIdType_720",
        IncludeChildrenDepartments: false,
        WithCustomData: false,
        WithIdentities: false,
        WithDepartmentIds: false,
    }
    response := client.SearchDepartmentMembers(&request)
    fmt.Println(response)
}

